src/Factory.ts

```diff
@@ -7,8 +8,11 @@ import {ExpressServer} from "./server/ExpressServer";
  * @param expressApp Express application instance
  * @param requireDirs Array of directories where from controller files will be loaded
  */
-export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[]): ControllerRunner {
+export function registerActionsInExpressApp(expressApp: any, requireDirs?: string[], container?: Container): ControllerRunner {
```

Comment:
I think this change is not recommended, since list of parameters can be become large very soon, so only first-priority params should be used (container cannot be called so). This can be accepted for now, but in the future I'm planning to change api of this method to make it more flexible.